pjnat64/src/pjnat64/pj-nat64.c

```diff
@@ -0,0 +1,1248 @@
+
+#include <pj/types.h>
+
+#include <pjsip-ua/sip_inv.h>
+#include <pjsip/sip_types.h>
+
+#include <pjsua.h>
+
+#include <pjnat64/pj-nat64.h>
+
+#define THIS_FILE "pj_nat64.c"
```

Comment:
`"pj-nat64.c"`

Reply:
We can replace it with `__FILE__` this will generally give an strict file name. Let's do that?

Reply:
No, we can't, because __FILE__ returns full path as I understand

Reply:
Ok. Renamed with use minus.